index.js

```diff
@@ -0,0 +1,14 @@
+$(document).ready(function(){
+
+  /*$(window).load(function(){
+     $('#imageID').fadeOut('slow',function(){
+           setTimeout(function(){
+              $(this).fadeOut('slow');*/
+
+              function practice (){
+
+              }
```

Comment:
If this is a placeholder for later, that's cool - but if it's not try removing unused code so that it's clear it's not a mistake.